zkevm-circuits/src/state_circuit/lexicographic_ordering.rs

```diff
@@ -97,7 +97,6 @@ pub struct Config {
     pub(crate) selector: Column<Fixed>,
     pub first_different_limb: BinaryNumberConfig<LimbIndex, 5>,
     limb_difference: Column<Advice>,
-    limb_difference_inverse: Column<Advice>,
```

Comment:
`limb_difference` is preserved, `limb_difference_inverse` was removed
because in state_circuit we relaxed situation for `limb_difference` might be equal to zero in `Rw::Padding`, For zero the invert do not exist, therefore removing `limb_difference_inverse`